### ISSUE TITLE
Create ractivejs/awesome-ractive repo

### ISSUE THREAD

**Issue by PaulMaly** [ractivejs/ractive#3158]
## Description:

Let's continue a discussion mentioned in this issue: https://github.com/ractivejs/ractive/issues/3156
I offer to create repo "awesome-ractive" to look for everything Ractive's stuff in one place, by analogy with other frameworks, like Vue, React and others. 

I'm ready to spend the time to curate this list. What do you, guys, think?

**Comment by ceremcem** [comment]
Well, I've heard about the repo by you and it seems quite nice. I've already created PR for the current repo. If the original creator won't have time, we might continue with any of the forks.

**Comment by ghost** [comment]
I still fail to see the need for a separate reference site. It will just cause more fragmentation and outdated references. It's not like the docs site is hard to update either. The MkDocs setup is totally optional, only needed if you're working on the theme or playground.

For content, all you will ever need is an editor with markdown preview (i.e. VS Code, PHP Storm, etc.) since the content is just vanilla markdown (having it easily editable by anyone without futzing with build steps and deps was the original intent of the revamp). The docs pages are even renderable by Github because of that: https://github.com/ractivejs/ractivejs.github.io/blob/dev/docs/integrations.md

<sub>Sure, GFM and Python Markdown may have differences but they're both supersets of the original Markdown which you're probably using 99% of the time anyways. Markdown flavor is nothing to worry about.</sub>

**Comment by PaulMaly** [comment]
@ceremcem Yep, but I believe that this list might be a part of Ractive's official repositories like Vue did.  

@fskreuz Maybe, but ["awesome-list"](https://github.com/sindresorhus/awesome) is a working standard with 70K+ stars on github and all other frameworks have this repo. As you can see, so many awesome things there, but Ractive's not ((((

I know that many people when choosing the framework for the new project, enters in search this simple phrase "awesome {{framework}}" and checks how many things community created for it. So I think that Ractive needs to use all possibilities to become more popular and it's one of.

**Comment by ceremcem** [comment]
> Yep, but I believe that this list might be a part of Ractive's official repositories like Vue did.

Meah... Both seems okay. 

> Maybe, but "awesome-list" is a working standard with 70K+ stars on github and all other frameworks have this repo. As you can see, so many awesome things there, but Ractive's not ((((

Good point. 70K+ stars means "Some people really needed it". This can not be simply ignored. 

@fskreuz This is like a clean Google results of "should I use / how should I use / how people use Ractive".

**Comment by ghost** [comment]
TL;DR: Really not a fan of fragmentation and duplication

The idea is great, I get the exposure aspect. But maintaining two (or more) instance of the same content is just more effort and maintenance. There are more pressing issues like keeping the docs accurate, having one canonical reference, fixing plugins, modernizing tools, tending to issues. Pointing this list to outdated docs, to non-working plugins, to outdated tools seems pointless. Adding more confusion to the already confusing confusion is confusing. 😄 

Fun fact: Going through the awesome list of awesome lists, it seems like some lists don't actually use the `awesome-` naming scheme (ES6 Tools, Julia, BEM, etc). You just need a list of curated things. Therefore it's likely possible that we can just create a page on the docs that serve as both an awesome list and a docs page. Two birds, one shotgun. 🤔 

Also, I see no difference landing on an awesome list from an "awesome" Google search vs landing on the docs page from an "awesome" Google search. If you're not landing on the docs site, we probably need better SEO or... docs site content. 😉

**Comment by PaulMaly** [comment]
@fskreuz 

> Really not a fan of fragmentation and duplication

That's called diversification and it's very healthy idea.)))

> But maintaining two (or more) instance of the same content is just more effort and maintenance.

Right, but look at Vue, for example. Their website has awesome documentation about the framework itself (btw Ractive's not) and just a links to external resources (including Awesome-Vue) right in the main menu. And it works, because, in fact, Vue's the young and resolute imitator of other frameworks including Ractive. Despite it, he kicks Ractive's @ss in most cases, especially in popularity.

So, we can just remove these pages from the official website and stop maintenance of this part. I believe it will release the time for work on documentation improvement. Frankly speaking, now these pages are useless long ago.

> Fun fact: Going through the awesome list of awesome lists, it seems like some lists don't actually use the awesome- naming scheme (ES6 Tools, Julia, BEM, etc). You just need a list of curated things. Therefore it's likely possible that we can just create a page on the docs that serve as both an awesome list and a docs page. Two birds, one shotgun.

Yep, you just have to correspond to the [awesome manifesto](https://github.com/sindresorhus/awesome/blob/master/create-list.md). But, you will lose all advantages, which gives this standard usage and diversification of resources. Btw, interesting that Awesome Vue has own website: http://www.awesome-vue.com/ and no one doesn't worry that it's a separate resource.
